Formula/a/aws-sam-cli.rb

```diff
@@ -410,6 +411,15 @@ def install
     venv.pip_install_and_link buildpath, build_isolation: false
 
     generate_completions_from_executable(bin/"sam", shell_parameter_format: :click)
+
+    # Remove binary built for a non-native architecture
+    if OS.linux?
+      arch = Hardware::CPU.arm? ? "arm64" : "x86_64"
+      rapid = libexec/"lib/python3.13/site-packages/samcli/local/rapid"
+      rapid.glob("aws-durable-execution-emulator-*").each do |file|
+        file.unlink unless file.basename.to_s.include?(arch)
```

Comment:
yeah, we should probably keep the binary, that is the whole feature release about.